domain-services/fiscal-event-service/src/main/java/org/egov/util/FiscalEventUtil.java

```diff
@@ -24,4 +33,41 @@ public AuditDetails enrichAuditDetails(String by, AuditDetails auditDetails, Boo
             return AuditDetails.builder().createdBy(auditDetails.getCreatedBy()).lastModifiedBy(by)
                     .createdTime(auditDetails.getCreatedTime()).lastModifiedTime(time).build();
     }
+
+    /**
+     * @param fiscalEventRequest
+     * @param jsonNode
+     */
+    public void enrichCoaDetails(FiscalEventRequest fiscalEventRequest, JsonNode jsonNode) {
```

Comment:
Don't name it jsonNode. Write a better name.

Reply:
done